src/Symfony/Bridge/PhpUnit/Tests/DeprecationErrorHandler/default.phpt

```diff
@@ -0,0 +1,63 @@
+--TEST--
+Test DeprecationErrorHandler in default mode
+--SKIPIF--
+<?php if (!file_exists('DeprecationErrorHandler.php')) print "skip"; ?>
```

Comment:
I suggest giving more details about the reason for the test being skipped (for instance ``skip can only run from a standalone install of the component``).